lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -2,19 +2,21 @@
     <ValidatedModelForm
         data-test-license-edit-form
         @model={{@manager.registration}}
-        @onSave={{@manager.onSave}}
-        @onError={{@manager.onError}}
+        @changeset={{@manager.changeset}}
+        @onSave={{action @manager.onSave}}
+        @onError={{action @manager.onError}}
```

Comment:
iirc, it's necessary to use the action helper anymore. as the `@action` decorator binds the action to the component class, with the ember >= 3.15.